helper-hardhat-config.js

```diff
@@ -50,8 +62,45 @@ const getNetworkIdFromName = async (networkIdName) => {
     return null
 }
 
+const autoFundCheck = async (contractAddr, networkName, linkTokenAddress, additionalMessage) => {
```

Comment:
The fund check shouldn't also do the funding. The fund check should just check if it can be funded. If yes, then it should run `autoFund`